src/olympia/lib/settings_base.py

```diff
@@ -1296,6 +1296,11 @@ def get_db_config(environ_var, atomic_requests=True):
             'level': logging.INFO,
             'propagate': False
         },
+        'celery.worker.strategy': {
+            'handlers': ['mozlog'],
+            'level': logging.WARNING,
```

Comment:
So this indicates that only logs with a level of `WARNING` or higher would be sent?

Reply:
Yes and we see in Stackdriver that the "Received..." log message is an INFO one.